addons/website_links/views/website_links_template.xml

```diff
@@ -6,6 +6,7 @@
                 <script type="text/javascript" src="/website_links/static/src/js/website_links.js"/>
                 <script type="text/javascript" src="/website_links/static/src/js/website_links_code_editor.js"/>
                 <script type="text/javascript" src="/website_links/static/src/js/website_links_charts.js"/>
+                <script type="text/javascript" src="/website_links/static/src/js/test_ui_flow.js"></script>
```

Comment:
And I would do the same as the above here and autoclose the script tag :)

Reply:
Done